AntennaTracker/control_scan.cpp

```diff
@@ -23,16 +23,21 @@ void Tracker::update_scan(void)
     }
 
     if (!nav_status.manual_control_pitch) {
-        float pitch_delta = g.scan_speed * 0.02f;
+        float pitch_delta = g.scan_speed_pitch * 0.02f;
         nav_status.pitch += pitch_delta * (nav_status.scan_reverse_pitch?-1:1);
-        if (nav_status.pitch < -90 && nav_status.scan_reverse_pitch) {
+        if (nav_status.pitch < g.pitch_min && nav_status.scan_reverse_pitch) {
             nav_status.scan_reverse_pitch = false;
         }
-        if (nav_status.pitch > 90 && !nav_status.scan_reverse_pitch) {
+        if (nav_status.pitch > g.pitch_max && !nav_status.scan_reverse_pitch) {
             nav_status.scan_reverse_pitch = true;
         }
         nav_status.pitch = constrain_float(nav_status.pitch, -90, 90);
```

Comment:
Should these 90s be `g.pitch_max` and `g.pitch_min`?

Reply:
fixed